tests/CMakeLists.txt

```diff
@@ -1,51 +1,82 @@
 # Copyright (c) 2015-2021, EPFL/Blue Brain Project
 # All rights reserved. Do not distribute without permission.
-# Responsible Author: Daniel.Nachbaur@epfl.ch
+# Responsible Authors: Daniel.Nachbaur@epfl.ch
+#                      Nadir Roman Guerrero <nadir.romanguerrero@epfl.ch>
 #
 # This file is part of Brayns <https://github.com/BlueBrain/Brayns>
 
-set(TEST_LIBRARIES brayns braynsIO braynsManipulators)
+include(CTest)
+enable_testing()
 
 configure_file(paths.h.in ${PROJECT_BINARY_DIR}/tests/paths.h)
 
-list(APPEND EXCLUDE_FROM_TESTS myPlugin.cpp)
+#list(APPEND CMAKE_MODULE_PATH ${OSPRAY_CMAKE_ROOT})
```

Comment:
We leave that commented or we can remove it ?

Reply:
Removed, thanks!